components/fetch-build-scan-data-cmdline-tool/build.gradle.kts

```diff
@@ -29,11 +35,32 @@ java {
     }
```

Comment:
Can you please check that the dependencies we pull in are the latest ones?

Reply:
✅ Done.

There is a openapi-generator-cli v6.0.0 that was released 11 days ago, but upgrading appears to break code generation. v5.4.0 is the latest 5.x version.

Reply:
cc @guylabs

Reply:
This is already been worked on: https://github.com/gradle/ge/issues/14349 Thanks for pining us!